code/modules/mob/living/simple_animal/hostile/regalrat.dm

```diff
@@ -314,7 +314,7 @@
 /// Turns a mouse into an angry mouse
 /datum/action/cooldown/riot/proc/convert_mouse(mob/living/basic/mouse/nearby_mouse)
 	// This mouse is already rat controlled, let's not bother with it.
-	if (istype(nearby_mouse.ai_controller, /datum/ai_controller/basic_controller/mouse/rat))
+	if (istype(nearby_mouse.ai_controller, /datum/ai_controller/basic_controller/mouse/rat) || nearby_mouse.stat == DEAD)
```

Comment:
Mice and roaches delete on death, is this necessary for those or only for frogs?